src/app.ts

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+import appRouter from "./routes/appRouter"; 
+
+const app = express();
+const port = 3000;
+
+app.use('/', appRouter);
+
+app.listen(port, () => {
+  console.log(`Server listening in port: ${port}`)
+})
```

Comment:
Since we are using TypeScript it's important to add types. 

`app.listen(port, (req : Request, res : Response) => {
res.send('ANY TEXT HERE')
}`